response_operations_ui/controllers/party_controller.py

```diff
@@ -68,13 +68,15 @@ def get_respondent_by_party_id(respondent_party_id):
 
 
 def survey_ids_for_respondent(respondent, ru_ref):
+    logger.info('Survey ids for respondent', ru_ref=ru_ref)
```

Comment:
I'd probably not bother with this log line - the code that calls the method that calls this method logs out `logger.info("Gathering data to view reporting unit", ru_ref=ru_ref)`, which feels good enough to me